libs/core/src/lib/busy-indicator/busy-indicator.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="fd-busy-indicator--l" *ngIf="loading" [attr.aria-label]="loadingLabel">
```

Comment:
I think component should provide a possibility to specify size of busy indicator via component interface (especially if we have classes like `fd-busy-indicator--l` and `fd-busy-indicator--m` coming from fundamental-styles),

Reply:
It has to be a certain size so that it fits inside of the box